packit/api.py

```diff
@@ -426,20 +429,30 @@ def status(self):
         else:
             logger.info("\nGitHub upstream releases: No releases found.")
 
+    async def status_get_builds(self, status):
         builds = status.get_builds()
         if builds:
             logger.info("\nLatest builds:")
             for build in builds:
-                koji_builds_str = "\n".join(f" - {b}" for b in builds[build])
-                logger.info(f"{build}:\n{koji_builds_str}")
+                logger.info(f"{build}:\n{builds[build]}")
         else:
             logger.info("There are no builds.")
 
+    async def status_get_updates(self, status):
         updates = status.get_updates()
         if updates:
             logger.info("\nLatest bodhi updates:")
             logger.info(tabulate(updates, headers=["Update", "Karma", "status"]))
 
+    async def status(self):
+        status = Status(self.config, self.package_config, self.up, self.dg)
+        t1 = asyncio.create_task(self.status_get_downstream_prs(status))
+        t2 = asyncio.create_task(self.status_get_dg_versions(status))
+        t3 = asyncio.create_task(self.status_get_up_releases(status))
+        t4 = asyncio.create_task(self.status_get_builds(status))
+        t5 = asyncio.create_task(self.status_get_updates(status))
```

Comment:
[asyncio.create_task()](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task) has been added in Python 3.7.
We want the code to be Python 3.6 compatible.